pkg/controller/controller_instance.go

```diff
@@ -964,21 +964,26 @@ func (c *controller) pollServiceInstance(instance *v1beta1.ServiceInstance) erro
 			return c.finishPollingServiceInstance(instance)
 		}
 
-		// We got some kind of error and should continue polling.
-		//
-		// The instance's Ready condition should already be False, so
-		// we just need to record an event.
 		reason := errorPollingLastOperationReason
 		message := fmt.Sprintf("Error polling last operation: %v", err)
-		glog.V(4).Info(pcb.Message(message))
```

Comment:
This log message was removed, but any polling error messages should continue to be logged.

Reply:
I will bring it back.

Reply:
done.